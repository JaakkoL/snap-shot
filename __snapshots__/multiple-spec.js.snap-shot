module.exports[`handles multiple snapshots 1`] = 1

module.exports[`handles multiple snapshots 2`] = 2

module.exports[`handles multiple snapshots in if/else 1`] = 1

module.exports[`multiple values 1`] = "first"

module.exports[`multiple values 2`] = "second"

module.exports[`multiple values 3`] = "third"

