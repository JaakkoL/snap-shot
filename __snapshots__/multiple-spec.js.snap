module.exports[`handles multiple snapshots 0`] = 1

module.exports[`handles multiple snapshots 1`] = 2

module.exports[`handles multiple snapshots in if/else 0`] = 1

